http.js

```diff
@@ -203,7 +203,7 @@ function start (entry, opts) {
     function disconnect () {
       clearInterval(interval)
       if (connected) {
-        emitter.removeListener('scripts:bundle', reloadScript)
+        emitter.removeListener('documents:index.html', reloadScript)
```

Comment:
This should probably be `documents:list`

Reply:
oh actually — nevermind; this doesn't seem to work the way I'd hoped it would :/